### ISSUE TITLE
Bluetooth Faulty

### ISSUE THREAD

**Issue by mchipeur** [feren-OS/Bug-Reporting-Center#157]
I am using Feren OS 2020.07 on my Lenovo Thinkpad Helix and I cannot get the Bluetooth to stay connected to my moiuse for more than 10 minutes. I then have to delete the pairing then scan and reconnect. I ensure it is "Trusted" and it does work for a few minutes but it invariably fails. 

`
Bus 001 Device 004: ID 0a5c:21e6 Broadcom Corp. BCM20702 Bluetooth 4.0 [ThinkPad]
`

**Comment by feren** [comment]
Does this still occur?

**Comment by mchipeur** [comment]
I started using a mouse with a dongle. I can try the Bluetooth mouse again.

I installed "Blueman" package. Now my Bluetooth audio devices seem to be
working.

**Comment by mchipeur** [comment]
I tried the original Bluetooth mouse and it worked without issue for a few
minutes. It was too much trouble to continue with it so I am sticking to
the dongle mouse.

**Comment by dominichayesferen** [comment]
Does this still occur on Feren OS November 2020 Snapshot?

**Comment by mchipeur** [comment]
I switched to a dongle based mouse after the frustration. (This Bluetooth
issue happened with all Linux distros I tried.)

I think my BT audio devices were behaving better towards the end, about a
month ago.

By "end" I mean that I switched back to Windows a month ago after being
unable to find sufficient work tools/apps to replace the Windows apps. The
web-based options were not satisfactory. Additionally, I have never had any
issues with Bluetooth on Windows. Ever.

Thank you for the follow up.
